WTWSMS/history/titles/c_plasencia.txt

```diff
@@ -6,15 +6,24 @@
 	liege = e_wre
 }
 470.1.1={
-	holder=20381
+	holder= 20381
 	liege = "k_visigoths"
 }
 480.1.1={
-	holder = 1301295 # Salla, Visigothic dux in Spain 483
+	holder= 20381
```

Comment:
Is it an intended change? 20381 is the holder since 470.

Reply:
I am not sure why that was changed